packages/contracts/mainnetDeployment/governanceSignatures.js

```diff
@@ -0,0 +1,50 @@
+// Define the permissioned function signatures
+const SET_STAKING_REWARD_SPLIT_SIG = (ethers.utils.id('setStakingRewardSplit(uint256)')).substring(0, 10);
```

Comment:
it might help to document the contract for each function and `setMaxFeeBps()` seems already removed

```
// CdpManager
const SET_STAKING_REWARD_SPLIT_SIG = ...
...
// CdpManagerStorage
const SET_GRACE_PERIOD_SIG = ...
...
// EBTCToken
const MINT_SIG = ...
...
// PriceFeed
const SET_FALLBACK_CALLER_SIG = ...
...
// BorrowerOperations & ActivePool
const SET_FEE_BPS_SIG = ...
...
// ActivePool & CollSurplusPool
const SWEEP_TOKEN_SIG = ...
...
// ActivePool
const CLAIM_FEE_RECIPIENT_COLL_SIG = ...
...
// BorrowerOperations & ActivePool
const SET_FEE_RECIPIENT_ADDRESS_SIG = ...
...
// Governor
const SET_ROLE_NAME_SIG = ...
...
// RolesAuthority
const SET_USER_ROLE_SIG = ...
...
// Ownable 
const TRANSFER_OWNERSHIP_SIG = ...
...
// AuthNoOwner & Auth
const SET_AUTHORITY_SIG = ...
...
```

Reply:
Good call, addressed.